products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,233 @@
+## Game of Thrones deaths
+
+This article is a part of a series intended to show how to use Memgraph
+on real-world data to retrieve some interesting and useful
+information.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [tutorial overview section](tutorials-overview.md).
+
+### Introduction
+
+**WARNING** - this tutorial could contain Game of Thrones ***spoilers***.  
+
+Game of Thrones is an American fantasy drama television series created by David Benioff and D. B. Weiss for HBO.
+It is an adaptation of A Song of Ice and Fire, George R. R. Martin's series of fantasy novels,  the first of which is
+A Game of Thrones. The Game of Thrones world is full of interesting characters, locations, scenarios, unexpected deaths, and plot twists.
+
+Even though, the COVID-19 pandemic hit the entire world in 2020 and is now starting to become one of the worst
+years in recent history, 2019 was also a huge disappointment to all the Game of Thrones fans. According to IMDb ratings, a seven-year 
+build-up resulted in a poorly written ending of the last season and ruined the ending of one of the most popular shows on the planet.  
+Nonetheless, if you want to know how many characters would have survived if Jon Snow had stayed dead, which House had the best Kill/Death Ratio
+or who was the biggest traitor in the show, you came to the right place!

```

Comment:
I see you've used the oxford comma before. That's great! You can use it here as well: `... Ratio, or who ...`